pkg/api/endpoints/util.go

```diff
@@ -34,42 +34,24 @@ import (
 // form for things like comparison.  The result is a newly allocated slice.
 func RepackSubsets(subsets []api.EndpointSubset) []api.EndpointSubset {
 	// First map each unique port definition to the sets of hosts that
-	// offer it.  The sets of hosts must be de-duped, using IP+UID as the key.
-	type addressKey struct {
-		ip  string
-		uid types.UID
-	}
+	// offer it.
 	allAddrs := map[addressKey]*api.EndpointAddress{}
 	portsToAddrs := map[api.EndpointPort]addressSet{}
 	for i := range subsets {
 		for j := range subsets[i].Ports {
```

Comment:
if you are going with a copy for epp, can you just say j, epp in the statement .